util/test/sub_test.py

```diff
@@ -1371,17 +1382,17 @@ def filter_errors(output_in, pre_exec_output, execgoodfile, execlog):
         continue # on to next test
 
     # c tests don't have a way to launch themselves
-    if is_c_test and chpllauncher != 'none':
+    if is_c_or_cpp_test and chpllauncher != 'none':
         sys.stdout.write('[Skipping c test: %s/%s]\n'%(localdir,test_filename))
```

Comment:
This and the if statement below say they are skipping C tests specifically.  I think it would be better to have a separate message when skipping a C++ test